src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -41,6 +58,10 @@
  * It creates a new object of ServiceCollection. This class performs CRUD operations for ServiceCollection.
  */
 public class ServiceCollection extends PersoniumCollection {
+    public interface IPersoniumServiceRoute {
+        public String getName();
```

Comment:
```suggestion
        String getName();
```

Checkstyle warns it is redundant public.